sql/src/main/java/io/crate/metadata/doc/DocTableInfoBuilder.java

```diff
@@ -161,13 +163,21 @@ private DocIndexMetaData buildDocIndexMetaDataFromTemplate(String index, String
         return partitions;
     }
 
+    private Collection<Reference> buildSysColumns() {
+        ArrayList<Reference> result = new ArrayList<>();
+        DocSysColumns.forTable(this.ident, (c,r) -> result.add(r));
+        return result;
+    }
```

Comment:
I think this could be moved to `DocSysColumn.forTable(Relationname) -> List<Reference>`